src/charts/bar.js

```diff
@@ -100,42 +102,64 @@ define(function(require) {
             isAnimated = false,
             ease = d3Ease.easeQuadInOut,
             animationDuration = 800,
-            interBarDelay = function(d, i) { return 70 * i},
+            interBarDelay = function(d, i) {
+                return 70 * i
+            },
 
             valueLabel = 'value',
             nameLabel = 'name',
 
             maskGridLines,
             baseLine,
+            reverseColorList = true,
 
             // Dispatcher object to broadcast the mouse events
             // Ref: https://github.com/mbostock/d3/wiki/Internals#d3_dispatch
             dispatcher = d3Dispatch.dispatch('customMouseOver', 'customMouseOut', 'customMouseMove'),
 
             // extractors
-            getName = ({name}) => name,
-            getValue = ({value}) => value,
-
-            _percentageLabelHorizontalX = ({value}) => xScale(value) + percentageLabelMargin,
-            _percentageLabelHorizontalY= ({name}) => yScale(name) + (yScale.bandwidth() / 2) + (percentageLabelSize * (3/8)),
-
-            _percentageLabelVerticalX = ({name}) => xScale(name),
-            _percentageLabelVerticalY = ({value}) => yScale(value) - percentageLabelMargin,
-
-            _percentageLabelHorizontalFormatValue = ({value}) => d3Format.format(horizontalLabelFormat)(value),
-            _percentageLabelVerticalFormatValue = ({value}) => d3Format.format(verticalLabelFormat)(parseFloat(value) * 100);
+            getName = ({
```

Comment:
I guess you have a linter that is unfolding these automatically? we prefer it as they were. Thanks!